modules/mauth-common/src/main/java/com/mdsol/mauth/util/MAuthHeadersHelper.java

```diff
@@ -1,28 +1,48 @@
 package com.mdsol.mauth.util;
 
+import com.mdsol.mauth.MAuthVersion;
+
 import java.util.UUID;
 
 public class MAuthHeadersHelper {
 
+  public static final String AUTH_HEADER_DELIMITER = ";";
```

Comment:
This is not a request to change but I just want to explain the reason behind adding `AUTH_HEADER_DELIMITER` in V2, as far as I understand.

Per [MAuth Protocol V2 Specification](https://docs.google.com/document/d/1X3X5V8LnNhKAqS53SfFJJnZe-TrjdMPw09tnO3ZpfdU/edit?usp=sharing):

> A semi-colon (";") to indicate the end of the digital signature (since semi-colons are not present in base64-encoding).  This is present so that in the future, the next version of the protocol could be specified on the same header after this semi-colon, e.g. MWSV2 <app_uuid>:<digital_signature_for_request>; MWSV3 <the_mwsv3_signature>;

I have no idea when we will have `MWSV3`, but if so, the signature will be added to the same `MCC-Authentication` header.

That is why I made a slightly different regex for V2 in Python (Removed `\A` and `\Z`, check for the token and `AUTH_HEADER_DELIMITER`):
https://github.com/mdsol/mauth-client-python/blob/0.2.1/mauth_client/consts.py#L13

But again, you can just kick the can 😄